libraries/api-documenter/src/yaml/YamlGenerator.ts

```diff
@@ -164,6 +171,55 @@ export class YamlGenerator {
     return yamlItem as IYamlItem;
   }
 
+  private _populateYamlMethod(yamlItem: Partial<IYamlItem>, docItem: DocItem): void {
+    const apiMethod: IApiMethod = docItem.apiItem as IApiMethod;
+    yamlItem.name = RenderingHelpers.getConciseSignature(docItem.name, apiMethod);
+
+    const syntax: IYamlSyntax = {
+      content: apiMethod.signature
+    };
+    yamlItem.syntax = syntax;
+
+    if (apiMethod.returnValue) {
+      syntax.return = {
+        type: [ apiMethod.returnValue.type ],
+        description: this._renderMarkdownFromDocElement(apiMethod.returnValue.description, docItem)
+      };
+    }
+
+    const parameters: IYamlParameter[] = [];
+    for (const parameterName of Object.keys(apiMethod.parameters)) {
+      const apiParameter: IApiParameter = apiMethod.parameters[parameterName];
+      parameters.push(
+        {
+           id: parameterName,
+           description:  this._renderMarkdownFromDocElement(apiParameter.description, docItem),
+           type: [ apiParameter.type || '' ]
+        } as IYamlParameter
+      );
+    }
+
+    if (parameters.length) {
+      syntax.parameters = parameters;
+    }
+
+  }
+
+  private _populateYamlProperty(yamlItem: Partial<IYamlItem>, docItem: DocItem): void {
+    const apiProperty: IApiProperty = docItem.apiItem as IApiProperty;
+
+    const syntax: IYamlSyntax = {
+      content: docItem.name + ': ' + apiProperty.type + ';' // TODO
```

Comment:
>apiProperty.type + ';' // TODO [](start = 37, length = 30)

this doesn't look complete

Reply:
That's what "TODO" means.  ;-)

It requires a *.api.json file format change, which I will do in a separate PR

---
In reply to: [137664022](https://github.com/Microsoft/web-build-tools/pull/363#discussion_r137664022) [](ancestors = 137664022)